visualizer/options-menu.css

```diff
@@ -4,7 +4,7 @@
   font-size: var(--small-text-size);
   position: absolute;
   z-index: 1;
-  background-color: rgba(var(--opposite-color-val), 0.85);
+  background-color: var(--options-menu-bg-color);
```

Comment:
Can we keep the translucency here: on some profiles the only frames added may be behind the options menu, and we want the user to be able to notice that something has changed.

(I have a plan to move the main filter controls out of the options menu and replace the circles in the key at the bottom with checkboxes, as a better solution to this problem, but that needs to be done after adding the dependency filters)

Reply:
Oh, I must have forgot to save the css file.
It looks like this on my machine:
`--options-menu-bg-color: rgba(0, 0, 0, 0.85);`

Anyway, I saw your next comment and will update this accordingly ;)